src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -55,19 +50,14 @@ function PageCard({
               onClick={() => history.push(`${pagesAndResourcesPath}/${page.id}/settings`)}
             />
           )}
-        </Card.Title>
-
-        <div className="mb-2"><StatusBadge status={page.enabled} /></div>
+          </Card.Title>
+          <StatusBadge status={page.enabled} />
+        </div>
 
-        <Card.Text className="flex-grow-1 m-0">
+        <Card.Text className="m-0">
           {page.description}
         </Card.Text>
 
-        {(page.allowedOperations.enable && !page.enabled) && (
```

Comment:
why has this been removed?

Reply:
Latest Figma design does not have enable button on PageCard and there is ticket to remove enable button also. 
https://openedx.atlassian.net/browse/TNL-8486
https://www.figma.com/file/SeNpRff5HxzdpCgoD5pmlQ/Apps-and-resources-landing-page?node-id=3055%3A1435

Reply:
Got it. There is no commit related to TNL-8486. Can you add it to the description?

Reply:
Sure.